### DISCUSSION TITLE
Poll Time for like the 8th time already

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [DEMOLITIONDON96/Demolition-Engine#83] [Polls]
What would you want to see in the next release?

**Comment by Theoyeah** [comment]
players noteskins..... :trollface:

**Reply by ghost** [reply]
da player of da note :trollface:

**Comment by ThatOneSillyGuyDon** [comment]
alrighty, player note skin it is, @Theoyeah mind porting your code over? since you seem to have got it working

**Reply by Theoyeah** [reply]
sorry i deleted, because it is so fucky, it uses mod folder + customs source notetypes skins won't work

**Reply by Theoyeah** [reply]
but if you want look at the lastest commit before i deleted and you can port it

**Reply by Theoyeah** [reply]
but you can use the code, go crazy
